browserstack.json

```diff
@@ -11,17 +11,33 @@
         "safari_previous",
         "opera_latest",
         "opera_previous",
-
         "edge_latest",
         "edge_previous",
         "ie_11",
         "ie_10",
-        "ie_9",
-
-        "mobile_safari_latest",
-        "mobile_safari_7_0",
-        "android_browser_latest",
-        "android_browser_4_4",
-        "ie_mobile_latest"
+        {
+            "os": "ios",
+            "os_version": "12.4",
+            "device": "iPhone XS",
+            "real_mobile": "true"
+        },
+        {
+            "os": "ios",
+            "os_version": "11.0",
```

Comment:
are we still supporting iOS 11?
latest version is 13 and the docs say "last two versions"
https://get.foundation/sites/docs/compatibility.html

Reply:
> iOS 7+ is actively supported but with some known bugs.

https://github.com/foundation/foundation-sites/blob/develop/.browserslistrc#L4

Reply:
11 is the lowest that we can still test afaik.

Otherwise we have to change it (breaking change) as many still need iOS 11 support (there were relevant issues).